articles/060_ros_middleware_interface.md

```diff
@@ -321,38 +297,50 @@ In order to ensure that these information are passed back to the same middleware
 
 The details of the interface necessary for the subscriber side are not (yet) described in this document.
 
+### Optionally exposing native handles
+
+The RMW interface only exposes middleware agnostic handles.
+But the middleware implementation can optionally provide additional API to provide native handles.
+E.g. for a given ROS publisher handle a specific implementation can provide an API to access publisher related handles specific to the implementation.
+
+While using such a feature would make the user land code specific to the middleware implementation it provides a way to use features of a middleware implementation which are not exposed through the ROS interface.
+
 ## Current implementation
 
 The described concept has been implemented in the following packages:
 
-- the package [rmw](https://github.com/ros2/rmw/tree/master/rmw) defines the middleware interface
+- The package [rmw](https://github.com/ros2/rmw/tree/master/rmw) defines the middleware interface.
 
-  - the functions are declared in [rms/rmw.h](https://github.com/ros2/rmw/blob/master/rmw/include/rmw/rmw.h)
-  - the handles are defined in [rmw/types.h](https://github.com/ros2/rmw/blob/master/rmw/include/rmw/types.h)
+  - The functions are declared in [rms/rmw.h](https://github.com/ros2/rmw/blob/master/rmw/include/rmw/rmw.h).
+  - The handles are defined in [rmw/types.h](https://github.com/ros2/rmw/blob/master/rmw/include/rmw/types.h).
 
-- the package [rmw_connext_cpp](https://github.com/ros2/rmw_connext/tree/master/rmw_connext_cpp) implements the middleware interface using [RTI Connext DDS](http://www.rti.com/products/dds/index.html) based on statically generated code
+- The package [rmw_fastrtps_cpp](https://github.com/ros2/rmw_fastrtps/tree/master/rmw_fastrtps_cpp) implements the middleware interface using [eProsima Fast-RTPS](http://www.eprosima.com/index.php/products-all/eprosima-fast-rtps) based on the introspection type support.
```

Comment:
This is the first time the term introspection is being used. Maybe moving out `rosidl_typesupport_introspection_cpp` from the connext dynamic sub-section would make more sense given that it's used by this package as well

Reply:
Done in e3f9be6c2075351b3ebd7f6c162744edb08a3dd0.